DQM/TrackerMonitorTrack/src/MonitorTrackResiduals.cc

```diff
@@ -38,6 +38,11 @@ void MonitorTrackResidualsBase<pixel_or_strip>::bookHistograms(DQMStore::IBooker
     m_cacheID_ = cacheID;
     this->createMEs( ibooker , iSetup);
   }
+  std::string topFolderName_ = "SiStrip";
+  SiStripFolderOrganizer folder_organizer;
+  folder_organizer.setSiStripFolderName(topFolderName_);
+  tkhisto_ResidualsMean = new TkHistoMap(ibooker , topFolderName_ ,"TkHMap_ResidualsMean", 0.0,true);
```

Comment:
Hi @dmitrijus and @davidlange6, thanks a lot for the suggestion. 

Consider that https://github.com/aspiezia/cmssw/blob/a0a714e6c65870a27ee5a346d1d0e75367573276/DQM/SiStripMonitorTrack/interface/SiStripMonitorTrack.h#L131 is not the code I am modifying. It was just an example showing that everywhere in cmssw, the tracker maps are defined as I am doing. 

Anyway, I have tried your suggestion in the code I am modifing and it does not compile because of this line: https://github.com/aspiezia/cmssw/blob/a0a714e6c65870a27ee5a346d1d0e75367573276/DQM/SiStripMonitorTrack/interface/SiStripMonitorTrack.h#L131 
Do you have a suggestion on how to change it?

Thanks,
Aniello